rust/README.md

```diff
@@ -13,6 +13,30 @@ Most other language implementations of Noosphere bind to this Rust
 implementation internally, so making changes to Noosphere for a package
 targetting another language will most likely involve writing some Rust.
 
+## Rust Components
+
+```mermaid
+flowchart TD
+
+noosphere((Noosphere))
+cli((CLI))
+
+noosphere --> api
+noosphere --> core
+noosphere --> fs
+noosphere --> storage
+noosphere --> into
+cli --> api 
```

Comment:
I think this isn't quite right; the `cli` depends most critically on `noosphere`.

Reply:
Nice catch!! Updated